packages/fast-components-react-msft/src/button/button.spec.tsx

```diff
@@ -113,6 +113,35 @@ describe("button", (): void => {
         expect(rendered.find("div.slotBefore").length).toBe(1);
     });
 
+    test("should render prop into the before content location if prop exist", () => {
+        const props: ButtonHandledProps = {
+            children: "Foo",
+            beforeContent: (classname?: string): React.ReactNode => {
+                return (
+                    <svg
+                        width="16"
```

Comment:
do we know if these values should be pixels? Not blocking...genuinely curious as they seem to differ place to place.

Reply:
These are correct being unitless. Where are you seeing px afterwards?